renderers/abc_renderer.js

```diff
@@ -4,7 +4,13 @@ const abc = require('abcjs');
 function renderAbc(str, opts) {
   const div = document.createElement('div');
   abc.renderAbc(div, str, {
-    visualTranspose: opts.transpose
+    visualTranspose: opts.transpose,
+    staffwidth: opts.maxWidth,
+    paddingtop: 0,
+    paddingbottom: 0,
+    paddingright: 0,
+    paddingleft: 0,
+    wrap: { minSpacing: 1.8, maxSpacing: 2.7 },
```

Comment:
What do the `1.8` and `2.7` mean?

Reply:
Pulled it out of their docs as the "recommended values":
https://github.com/paulrosen/abcjs/blob/master/docs/api.md